Bottom: c6cf65fe1374a256d053979c58505516d89fa6ce
Top:    a0d620c83a5f202817b4ad05221e95008e34a381
Author: Bart Van Assche <bvanassche@acm.org>
Date:   2013-07-28 11:19:48 +0200

[3/4] rsocket: Remove the unused variable 'ret'

The variable 'ret' is assigned a value but that value is never used.
This triggers the following compiler warning:

src/rsocket.c:3720:9: warning: variable 'ret' set but not used [-Wunused-but-set-variable]

Hence remove this variable.

Signed-off-by: Bart Van Assche <bvanassche@acm.org>


---

diff --git a/src/rsocket.c b/src/rsocket.c
index abdd392..ef24dfb 100644
--- a/src/rsocket.c
+++ b/src/rsocket.c
@@ -3717,11 +3717,11 @@ static void rs_svc_process_sock(void)
 static uint8_t rs_svc_sgid_index(struct ds_dest *dest, union ibv_gid *sgid)
 {
 	union ibv_gid gid;
-	int i, ret;
+	int i;
 
 	for (i = 0; i < 16; i++) {
-		ret = ibv_query_gid(dest->qp->cm_id->verbs, dest->qp->cm_id->port_num,
-				    i, &gid);
+		ibv_query_gid(dest->qp->cm_id->verbs, dest->qp->cm_id->port_num,
+			      i, &gid);
 		if (!memcmp(sgid, &gid, sizeof gid))
 			return i;
 	}
